data/mods/Aftershock/items/books.json

```
@@ -39,7 +39,7 @@
     "id": "afs_welcome2_book",
     "type": "BOOK",
     "name": { "str": "Unofficial History of Salus IV Vol. 2", "str_pl": "copies of Unofficial History of Salus IV Vol. 2" },
-    "description": "Even during the Discontinuity Salus IV's population continued to drop as there were easier places to live even in the Salus system, from honeycombed asteroid habitats, to domed cities with gorgeous views.  As with many other governments during the Discontinuity the Interstellar Soviet collapsed into a vipers pit of microstates, solar corporations and other forms of self governance.  During this time Mercurial Genomics a corporation specializing in Uplifts successfully laid claim to Salus IV in the court system set up by ESCA.  The Whately family, which runs Mercurial Genomics begins leasing development and exploitation permits and allows the establishment of minor UICA outposts within Salus IV.  There is an official citizen count of less than fifty thousand sentient beings mostly in orbit of Salus IV.  It is believed however that there are at least that many non-citizen sentients spread amongst squatter, smuggler, pirate, and feral uplift communities.",
+    "description": "Even during the Discontinuity Salus IV's population continued to drop as there were easier places to live even in the Salus system, from honeycombed asteroid habitats, to domed cities with gorgeous views.  Mercurial Genomics was an early member of the nascent post-Discontinuity government efforts.  As a result of this member they have successfully reclaimed to Salus IV in the court system set up by ESCA.  The Whately family, which runs Mercurial Genomics begins leasing development and exploitation permits and allows the establishment of minor UICA outposts within Salus IV.  There is an official citizen count of less than twenty thousand sentient beings mostly in orbit of Salus IV.  It is believed however that there are at least that many non-citizen sentients spread amongst squatter, smuggler, pirate, and feral uplift communities.",
```

```suggestion
    "description": "Even during the Discontinuity Salus IV's population continued to drop as there were easier places to live even in the Salus system - from honeycombed asteroid habitats to domed cities with gorgeous views.  Mercurial Genomics was an early member of the nascent post-Discontinuity government efforts.  As a result of this membership, they have successfully reclaimed Salus IV in the court system set up by ESCA.  The Whately family, which runs Mercurial Genomics, began leasing development and exploitation permits and allows the establishment of minor UICA outposts within Salus IV.  There is an official citizen count of less than twenty thousand sentient beings, albeit mostly in orbit.  It is believed, however, that there are at least that many non-citizen sentients spread amongst squatter, smuggler, pirate, and feral uplift communities.",
```
Sentients or sapients?

---

sapients thanks!